jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -691,7 +691,7 @@
                 export DNS_ZONE_NAME="k8s-federation-1-4.com."
                 export FEDERATIONS_DOMAIN_MAP="federation=k8s-federation-1-4.com."
                 export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the clusters will be created. Federation components are now deployed to the last one.
-                export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+                export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-f8n-1-4"
```

Comment:
Use $PROJECT instead?  It seems that would have avoided the bug in the first place?